artifacts/source_type.py

```diff
@@ -40,6 +40,13 @@ def type_indicator(self):
           u'Invalid source type missing type indicator.')
     return self.TYPE_INDICATOR
 
+  @property
+  def attributes_definition(self):
```

Comment:
Also no reason.  I was actually going to implement it as a method at first and don't remember why I didn't.